experimental/traffic-portal/src/app/api/testing/profile.service.ts

```diff
@@ -215,11 +228,39 @@ export class ProfileService {
 	public async deleteProfile(id: number | ResponseProfile): Promise<ResponseProfile> {
 		const index = this.profiles.findIndex(t => t.id === id);
 		if (index === -1) {
-			throw new Error(`no such Type: ${id}`);
+			throw new Error(`no such profile: ${id}`);
 		}
 		return this.profiles.splice(index, 1)[0];
 	}
 
+	/**
+	 * Export Profile object from the API.
+	 *
+	 * @param id Specify unique identifier (number) of a specific Profile to retrieve the export object.
+	 * @returns The requested Profile as attachment.
+	 */
+	public async exportProfile(id: number | ResponseProfile): Promise<ProfileExport> {
+		const index = this.profiles.findIndex(t => t.id === id);
+		if (index === -1) {
+			throw new Error(`no such Profile: ${id}`);
+		}
+		return this.profileExport;
+	}
```

Comment:
this method will always throw an error when passed a `ResponseProfile`, because the types `number` and `ResponseProfile` will never compare as equal using `===`.